products/extensions/subscriptions/guides/mixed-checkout.md

```diff
@@ -0,0 +1,164 @@
+---
+nav:
+  title: Mixed checkout
+  position: 20
+
+---
+# Mixed subscription checkout
+
+::: info
+Available since Shopware version 6.7.4.0
+:::
+
+This guide describes how the mixed carts in the subscription feature works and how extensions should integrate with it.
+Mixed carts let customers buy subscription products and one‑time products in the same checkout while keeping subscription calculation isolated and predictable.
+
+Please familiarise yourself with the [concept](../concept.md) first before continuing here.
+
+## Overview
+
+Subscription line items are ordinary product line items in the main cart, but they carry subscription plan and interval IDs in their payload.
+During cart calculation, line items containing subscription plan and interval IDs in their payload are collected and grouped by plan and interval.
+For each group a derived _managed_ subscription context and a _managed_ subscription cart are created and calculated using the subscription cart calculation path.
+These represent the context and content of the later generated subscription.
+
+Managed subscription contexts and carts are persisted in the database as well.
+They are linked back to the main context by the `subscription_cart` database table.
+
```

Comment:
A little bonus, if you're intersted: 
A real cherry on top could be an illustration here: 

Shopware (Parent) Context 1:n Subscription Managed Context with showing the extension Map and such
Maybe ask the docs team or come up with something on your own.